README.md

```diff
@@ -25,11 +33,47 @@ Teaclave (incubating)](https://teaclave.apache.org/).
   - [Run Rust Applications](#run-rust-applications)
     - [Run Rust Applications in QEMUv8](#run-rust-applications-in-qemuv8)
     - [Run Rust Applications on other platforms](#run-rust-applications-on-other-platforms)
+  - [Test](#test)
 - [Documentation](#documentation)
 - [Publication](#publication)
 - [Contributing](#contributing)
 - [Community](#community)
 
+
+## TA Development Modes
+
+### Comparison
+
+#### `no-std`
+
+- **Pros**:
+  - Reuses standard Rust AArch64 tier-1 toolchain target (`aarch64-unknown-linux-gnu`).
```

Comment:
This is true for 32-bit systems (AArch32) as well where toolchain target is `arm-unknown-linux-gnueabihf`.

Reply:
done